docs/source/starters/new_project_tools.md

```diff
@@ -1,4 +1,199 @@
-# Configure a new project
+# Using tools to configure your Kedro project
 
-<!--TO DO-->
-<!--Detailed usage of add-ons goes here-->
+## Configuring your tool selection
+
+There are several ways to configure your project tool when creating a new Kedro project.
+
+### Through the `kedro new` prompt
+
+Navigate to the directory in which you would like to create your new Kedro project, and run the following command:
+
+```bash
+kedro new
+```
+
+This will start the new project creation wizard<!--wording-->, promptng you to input a project name, any tools you would like to include, and whether or not you would like to populate the project with example code. The creation wizard will provide you with a list of available tools and a brief description of each:
```

Comment:
```suggestion
This will start the new project creation wizard<!--wording-->, prompting you to input a project name, any tools you would like to include, and whether or not you would like to populate the project with example code. The creation wizard will provide you with a list of available tools and a brief description of each:
```